patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch

```diff
@@ -25,5 +25,5 @@ index aa32f0de5..289684816 100644
  				return;
  
 -- 
-2.25.0
+2.17.1
```

Comment:
You really should upgrade 😂

Reply:
![image](https://user-images.githubusercontent.com/5572/77675530-57a93400-6f8d-11ea-84a0-37a825cdc979.png)
:tada: